package.json

```diff
@@ -49,9 +49,9 @@
     }
   },
   "browserslist": [
-    "> 1%",
-    "last 2 versions",
-    "not ie <= 8"
+    "last 1 Firefox versions",
```

Comment:
How about this one:
https://browserl.ist/?q=%3E+0.5%25%2C+last+2+Firefox+versions%2C+Firefox+ESR%2C+not+ie+%3C+12

Benefit: No explicit allowlist, but still excludes IE (we do want to exclude IE, right?)

Reply:
I'd remove opera mini. I think they don't even support `fetch`. Maybe we publish a compat version at some point. But I bet if fully breaks because of CSS anyway.

Reply:
Yaiks, I assumed it was Blink.